src/routes/v2/authenticated/review.ts

```diff
@@ -185,6 +191,21 @@ export class ReviewsRouter {
       description
     )
 
+    // Step 5: Create notifications
+    collaborators.forEach(async (user: User & { SiteMember: SiteMember }) => {
+      // Don't send notification to self
+      if (user.id.toString() === userWithSiteSessionData.isomerUserId) return
+      const notificationType = reviewersMap[user.email || ""]
+        ? "sent_request"
+        : "request_created"
+      await this.notificationsService.create({
+        siteMember: user.SiteMember,
+        link: "TODO",
+        notificationType,
+        notificationSourceUsername: userWithSiteSessionData.email,
+      })
+    })
```

Comment:
could we use `Promise.all(collaborators.map(...))` instead? see [here](https://gist.github.com/joeytwiddle/37d2085425c049629b80956d3c618971) for how `forEach` behaves in async loops (badly)

Reply:
Good catch, I forgot to swap these out - fixed in b9449d4